velox/functions/prestosql/aggregates/ApproxDistinctAggregate.cpp

```diff
@@ -201,6 +205,7 @@ class ApproxDistinctAggregate : public exec::Aggregate {
         }
 
         auto group = groups[row];
+        RowSizeTracker t(group + rowSizeOffset_, *allocator_);
```

Comment:
- t -> tracker
- I noticed that there is incrementRowSize method in Aggregate.h which is not used. It would be nice to remove it.
- It would be helpful to add some comments in Aggregate.h to explain that aggregate functions must use this mechanism to track the size of the accumulator. Perhaps, put this into comments for isFixedSize.
- As written, this is a bit verbose. I wonder if there is a way to shorten this code. Maybe: `RowSizeTracker tracker(group, this);`